mozetl/shield/containers.py

```diff
@@ -0,0 +1,34 @@
+from pyspark.sql.types import StringType, LongType
+
+from ..basic import convert_pings, DataFrameConfig
+from .utils import testpilot_etl_boilerplate
+
+
+def transform_pings(sqlContext, pings):
+    return convert_pings(
+        sqlContext,
+        pings,
+        DataFrameConfig([
+            ("uuid", "payload/payload/uuid", None, StringType()),
+            ("userContextId", "payload/payload/userContextId", None, StringType()),
+            ("clickedContainerTabCount",
+             "payload/payload/clickedContainerTabCount", None, LongType()),
+            ("eventSource", "payload/payload/eventSource", None, StringType()),
+            ("event", "payload/payload/event", None, StringType()),
+            ("hiddenContainersCount", "payload/payload/hiddenContainersCount", None, LongType()),
+            ("shownContainersCount", "payload/payload/shownContainersCount", None, LongType()),
+            ("totalContainersCount", "payload/payload/totalContainersCount", None, LongType()),
+            ("totalContainerTabsCount",
+             "payload/payload/totalContainerTabsCount", None, LongType()),
+            ("totalNonContainerTabsCount",
+             "payload/payload/totalNonContainerTabsCount", None, LongType()),
+            ("pageRequestCount", "payload/payload/pageRequestCount", None, LongType()),
+            ("test", "payload/test", None, StringType()),
+        ], lambda ping: ping['payload/test'] in ["@testpilot-containers", "@shield-study-containers"])
```

Comment:
I know we've discussed this before, but I can't remember the conclusion. Do we actually want shield and testpilot data in one table? Will we be comparing testpilot and shield users or will we focus our analysis on the differences between experiment and control within the shield study?